### PR TITLE
engine: Hide feedback in Toc at the bottom of the page

### PR DESCRIPTION
**Pull request by JakeSCahill**
When users reach the bottom of the page on desktop, we hide the feedback thumbs in the ToC to avoid displaying two copies of them: one in the footer, and one in the ToC.

### CODE REVIEW

**Review by Deflaimun — APPROVED** [review]
LGTM

File: src/theme/DocItem/Layout/index.js [hunk 1]
```diff
@@ -195,6 +195,22 @@ export default function DocItemLayout({ children }) {
   const { editUrl } = metadata;
   const [show, setShow] = useState(false);
   const [positiveFeedback, setPositiveFeedback] = useState(true);
+  // Hide the feedback thumbs in the Toc when the user reaches the bottom of the page.
+  useEffect(() => {
+    document.addEventListener('scroll', function(e){
+      let documentHeight = document.body.scrollHeight;
+      let currentScroll = window.scrollY + window.innerHeight;
+      // Wait until the user is [modifier]px from the bottom.
+      let modifier = 300;
```
Comment (shorgi) [inline src/theme/DocItem/Layout/index.js:204]: Future improvement: is 300 the height of the footer? Can we programmatically get the height of the footer?

**Review by shorgi — APPROVED** [review]
Nice! LGTM

File: src/theme/DocItem/Layout/index.js (hunk 1 above)
Comment (JakeSCahill) [reply src/theme/DocItem/Layout/index.js:204]: I thought about this, but couldn't think of an easy answer and this seemed to work quite well. We should certainly update this in future when we have a better solution 😄

Comment (Deflaimun) [reply src/theme/DocItem/Layout/index.js:204]: Good catch. Depending on the screen size, both appear
image
Tested screen size 1200x1100

**Pull request merged**